scenes/src/app/scenes/slider/slider-scene.ts

```diff
@@ -13,7 +13,7 @@ export class SliderScene implements AfterViewInit {
   @ViewChild('volume') volume!: MatSlider;
 
   ngAfterViewInit() {
-    this.volume.focus();
+    (this.volume as any).focus();
```

Comment:
Is this as any necessary?  `MatSlider` should have a `focus` method.

Reply:
Yeah I don't understand this. I don't actually see any `focus` method on the class or in the compiled .d.ts?

https://github.com/angular/components/blob/6011b7d7e789fd9ad0c724df20501c2758f750e2/src/material/slider/slider.ts#L95

Reply:
I think it might be because of recent breaking changes to the slider for Angular v15. Basically a whole rework. Maybe add a TODO here and keep the cast for now?

Reply:
I've added the TODO